java/java.editor/src/org/netbeans/modules/java/editor/options/InlineHintsOptionsPanelController.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.java.editor.options;
+
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+import javax.swing.JComponent;
+import org.netbeans.spi.options.OptionsPanelController;
+import org.openide.util.HelpCtx;
+import org.openide.util.Lookup;
+
+public final class InlineHintsOptionsPanelController extends OptionsPanelController {
+    
+    private InlineHintsPanel panel;
+    
+    private final PropertyChangeSupport pcs = new PropertyChangeSupport(this);
+                    
+    @Override
+    public void update() {
+        panel.load( this );
+    }
+    
+    @Override
+    public void applyChanges() {
+        panel.store();
+    }
+    
+    @Override
+    public void cancel() {
+	// need not do anything special, if no changes have been persisted yet
+    }
+    
+    @Override
+    public boolean isValid() {
+        return true; // Always valid 
+    }
+    
+    @Override
+    public boolean isChanged() {
+	return panel.changed();
+    }
+    
+    @Override
+    public HelpCtx getHelpCtx() {
+	return new HelpCtx("netbeans.optionsDialog.java.inlineHints");
```

Comment:
indentation is missing in a few places in this class. Possibly an issue with tab/spaces if its copy pasted.

Reply:
It is a copy from the `Controller` in the same directory. Actually your settings might be different. If you look at the changeset on github everything is in line and the same was true on my system (default NetBeans settings). Anyway, I ran the class through the reformatter and it will be updated with spaces.

Reply:
wait, you say you don't see it like this:
![image](https://user-images.githubusercontent.com/114367/204954628-c63850ec-60a5-4f90-a03e-13127f5aaf0d.png)

Reply:
This is the result of a clean build from NetBeans:

![image](https://user-images.githubusercontent.com/2179736/205146655-e8920b5b-f498-42c0-a52b-fd59b7663453.png)

The same on Github (the marked area is 8 chars wide):

![image](https://user-images.githubusercontent.com/2179736/205147271-bacd3e1c-224f-4d69-97cd-ec3d213dc3e4.png)

The method declaration (`public HelpCtx`) was idented with 4 spaces, the method body (`return new HelpCtx`) is idented by a single tab (equivalent to 8 spaces).

Reply:
I see the issue now. This github account was set to render tabs with 4 spaces by default. Feel free to ignore this, however, i think it would still be better for new code to not use tabs at all. Since if they are mixed with spaces like here, it might lead to inconsistent formatting.